widget/calendar.go

```diff
@@ -43,30 +49,49 @@ type Calendar struct {
 // Since: 2.6
 func NewCalendar(cT time.Time, changed func(time.Time)) *Calendar {
 	c := &Calendar{
-		currentTime: cT,
-		OnChanged:   changed,
+		displayedMonth: cT,
+		OnChanged:      changed,
 	}
 
 	c.ExtendBaseWidget(c)
 	return c
 }
 
+// SetDisplayedDate sets the currently displayed year and month
+func (c *Calendar) SetDisplayedDate(date time.Time) {
```

Comment:
Isn't the current /display/ at month granularity not date granularity?

Reply:
Yes, `SetDisplayedMonth` seems more idiomatic